Sources/SwiftGen/main.swift

```diff
@@ -47,6 +47,8 @@ let main = Group {
     }
   }
 
+  $0.addCommand("xcfilelist", "generates xcfilelists based on the configuration file ", XCFileListCLI.generate)
```

Comment:
```suggestion
  $0.addCommand("xcfilelist", "generates xcfilelists based on the configuration file", XCFileListCLI.generate)
```

Not sure if this should be renamed to something like `generate-xcfilelists` to clarify it's purpose. Or a command + subcommand such as `xcfilelist generate`? To better match the structure of the other groups.